### DISCUSSION TITLE
Rejection tracing for axum_extra

### DISCUSSION THREAD

**Discussion by 39george** [tokio-rs/axum#2525] [Q&A]
### Summary

I am using the env_filter feature in my backend application to display all rejections. I have added the following directive to achieve this:
```rust
.add_directive("axum::rejection=trace".parse().unwrap())
```
This directive works well for all axum::extract::* types, but it seems that axum_extra::extract::Query does not trace rejections to the terminal.
My questions are:
Is rejection tracing implemented for axum_extra?
If yes, how can I enable it?
Thank you for your help!

### axum version

0.7.4

**Comment by davidpdrsn** [comment]
No I don’t think it is but totally should be! Do you wanna make issue about it?

**Reply by 39george** [reply]
Thank you for your response! I will go ahead and create an issue for this.
